### PR TITLE
Fix 7 e 5 create table vacancies coder vacancies viktoriia zakharko

### PR DESCRIPTION
**Pull request by Viktorika-p**

### CODE REVIEW

**Comment by okslisovska** [comment]
This pull request can not be approved by me due to the following issues:
1. The changes in settings.py are very suspicious. 
2. There are some issues with imports.
3. There is no need for the coder_vacancies table. Django has ManyToManyField() for it. We have it on our DB schema just to represent many-to-many relationship.

File: codeArena/codeArena/settings.py [hunk 1]
```diff
@@ -21,10 +18,10 @@
 # See https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = config('SECRET_KEY')
+SECRET_KEY = 'erofeorpegiheigeporg'
```
Comment (s0fko) [inline codeArena/codeArena/settings.py:21]: This changes shouldn't be committed.

For using and understanding local variables please have a look at [Decouple](https://pypi.org/project/python-decouple/) library and create .env file with them. We imported this library at 11 string and `config()` is it's method, which imports variables from .env file

Please change this thing everywhere, where `config()` method is using

File: codeArena/codeArena/settings.py [hunk 2]
```diff
@@ -40,8 +37,9 @@
     'django.contrib.staticfiles', 
     'rest_framework',
     'task',
-    'competition',
```
Comment (s0fko) [inline codeArena/codeArena/settings.py:43]: 'competition' app was created by me and you shouldn't delete it from here

If you haven't this changes in your local repository, than please pull changes from GitHub

File: codeArena/codeArena/settings.py [hunk 3]
```diff
@@ -40,8 +37,9 @@
...
     'user',
+    'vacancies',
+
```
Comment (s0fko) [inline codeArena/codeArena/settings.py:42]: Please remove empty string, according to [pep8](https://www.python.org/dev/peps/pep-0008/)

File: codeArena/codeArena/settings.py [hunk 4]
```diff
@@ -151,4 +149,4 @@
 ALLOWED_HOSTS = ['127.0.0.1']
 
 DATABASE_ROUTERS = ['routers.db_routers.PostgresRouter',
-                    'routers.db_routers.MongoRouter']
+                    'routers.db_routers.MongoRouter']
```
Comment (s0fko) [inline codeArena/codeArena/settings.py:152]: Please add newline to the end of file

File: codeArena/vacancies/models.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+from user.enums import UserStatus
+from django.utils.timezone import now
+
+
+
+class Vacancies(models.Model):
+    name = models.CharField(max_length=30)
+    description = models.CharField(max_length=250)
+    recruiter_id = models.IntegerField(max_length=250)
+    company = models.CharField(max_length=20)
```
Comment (s0fko) [inline codeArena/vacancies/models.py:13]: I guess we need more than 20 characters for company names

File: codeArena/vacancies/models.py [hunk 6]
```diff
@@ -0,0 +1,30 @@
+from codeArena.user.models import Coder
+from django.db import models
+from django.db.models.fields import CharField, DateField, FloatField, IntegerField, EmailField, DateTimeField, ImageField
+from user.enums import UserStatus
+from django.utils.timezone import now
+
```
Comment (s0fko) [inline codeArena/vacancies/models.py:6]: Please add only 2 blank lines between the import statements and other code

File: codeArena/vacancies/models.py [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    recruiter_id = models.IntegerField(max_length=250)
+    company = models.CharField(max_length=20)
+    created_at = models.DateTimeField(default=now, editable=False)
+    updated_at = models.DateTimeField(null=True, blank=True)
+    language_id = models.IntegerField(max_length=20)
+    company_logo = models.ImageField(Null=True)
+
+    def __str__(self):
+        return f'{self.name}, {self.description}, {self.recruiter_id}, {self.company}, {self.created_at}, {self.updated_at}, {self.language_id}, {self.company_logo}'
+
```
Comment (s0fko) [inline codeArena/vacancies/models.py:21]: Please add only 2 blank lines between the classes

File: codeArena/vacancies/models.py [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+
+
+
+class Coder_vacancies(models.Model):
```
Comment (s0fko) [inline codeArena/vacancies/models.py:25]: Please use the CapWords convention for class names according to [pep8](https://www.python.org/dev/peps/pep-0008/#class-names)

So, your class name should be named `CoderVacancies` instead

**Review by s0fko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**